src/py/flwr/server/superlink/fleet/message_handler/message_handler.py

```diff
@@ -124,5 +127,17 @@ def get_run(
             fab_id=run.fab_id,
             fab_version=run.fab_version,
             override_config=user_config_to_proto(run.override_config),
+            fab_hash=run.fab_hash,
         )
     )
+
+
+def get_fab(
+    request: GetFabRequest, ffs: Ffs  # pylint: disable=W0613
+) -> GetFabResponse:
+    """Get FAB."""
+    if result := ffs.get(request.hash_str):
+        fab = Fab(request.hash_str, result[0])
+        return GetFabResponse(fab=fab_to_proto(fab))
+
+    return GetFabResponse()
```

Comment:
Should we start to return an error code alongside the response from `message_handler` functions?